mobile/lib/services/native_api.dart

```diff
@@ -685,7 +679,7 @@ class ShareService {
     //   var imgFile = File('$directory/airqo_analytics_card.png');
```

Comment:
we need to minimise commented out code. Unless it is a temporary disable of some functionality...

Reply:
this is temporarily disabled functionality for sharing @Baalmart